network-api/networkapi/buyersguide/factory.py

```diff
@@ -219,38 +220,63 @@ class Meta:
         model = BuyersGuidePage
 
 
-class SoftwareProductPageFactory(PageFactory):
+class ProductPageVotesFactory(DjangoModelFactory):
 
     class Meta:
-        model = SoftwareProductPage
+        model = ProductPageVotes
+
+
+    vote_bins = LazyFunction(lambda: ','.join([str(randint(1,50)) for x in range(0,5)]))
+
+
+
+class ProductPageFactory(PageFactory):
+
+    class Meta:
+        model = ProductPage
 
     title = Faker('sentence')
-    handles_recordings_how = Faker('sentence')
-    recording_alert = LazyFunction(get_extended_yes_no_value)
-    recording_alert_helptext = Faker('sentence')
-    medical_privacy_compliant = Faker('boolean')
-    medical_privacy_compliant_helptext = Faker('sentence')
-    host_controls = Faker('sentence')
-    easy_to_learn_and_use = Faker('boolean')
-    easy_to_learn_and_use_helptext = Faker('sentence')
-    handles_recordings_how = Faker('sentence')
-    recording_alert = LazyFunction(get_extended_yes_no_value)
-    recording_alert_helptext = Faker('sentence')
-    medical_privacy_compliant = Faker('boolean')
-    medical_privacy_compliant_helptext = Faker('sentence')
-    host_controls = Faker('sentence')
-    easy_to_learn_and_use = Faker('boolean')
-    easy_to_learn_and_use_helptext = Faker('sentence')
+
+    privacy_ding = Faker('boolean')
+    adult_content = Faker('boolean')
+    uses_wifi = Faker('boolean')
+    uses_bluetooth = Faker('boolean')
+    company = Faker('company')
+    blurb = Faker('sentence')
+    product_url = Faker('url')
+    price = LazyAttribute(lambda _: randint(49, 1500))
+    worst_case = Faker('sentence')
+
     first_published_at = Faker('past_datetime', start_date='-2d', tzinfo=timezone.utc)
     last_published_at = Faker('past_datetime', start_date='-1d', tzinfo=timezone.utc)
 
+    # TODO: add image binding
```

Comment:
Should we add images in at this time? We landed some work that fixes the images in the template, but adding them here in the factory would be amazing. This is non blocking imo as it's just factory code, so if we don't add it in now let's make sure we open a ticket for it.

Reply:
let's make that part of https://github.com/mozilla/foundation.mozilla.org/issues/5777